core-web/libs/dotcms-scss/angular/_variables.scss

```diff
@@ -117,4 +117,8 @@ $tag-radius-md: 7px;
 $device-list-item-height: 3.625rem;
 $device-selector-grid-width: 31.49rem;
 $device-list-item-media-height: 2.5rem;
-$device-selector-header-height: 3.625rem;
+$device-selector-header-height: 2.625rem;
+
+// Page Tools
+$page-tools-width: 26.3rem;
+$page-tools-logo-size: 2.6rem;
```

Comment:
If you are using this only in those components store those variables there if they're are using more than one.

We need to clean this file for only global variables.